docs/reference/indices/apis/reload-analyzers.asciidoc

```diff
@@ -69,4 +69,33 @@ reload to ensure the new state of the file is reflected everywhere in the cluste
 POST /my_index/_reload_search_analyzers
 --------------------------------------------------
 // CONSOLE
-// TEST[s/^/PUT my_index\n/]
\ No newline at end of file
+// TEST[continued]
+
+The reload request returns information about the nodes it was executed on and the
+analyzers that were reloaded:
+
+[source,js]
+--------------------------------------------------
+{
+    "_shards" : {
+        "total" : 2,
+        "successful" : 2,
+        "failed" : 0
+    },
+    "reloaded_nodes" : [
```

Comment:
Yes, the format of the response here is a bit different. The reason for this ist that although reloading is centered around indices, it needs to be executed on each node an index shard might live on (and not only once on each shard in a replica set). So this is potentially more than one node per index. While I think the list of nodeIds that were part of reloading for each index is not that important to the average user, it can provide valuable debug information e.g. for UI clients later on.

Regarding your second point: I know we use index names as keys in several places but if not necessary I conciously try to avoid it. I makes parsing of responses more difficult because the parser need to deal with an unbounded set of keys as field names. While we mange to get around it in the places you mentioned, I try to avoid it if possible in order to make the rest response easier to parse.

Reply:
> although reloading is centered around indices, it needs to be executed on each node an index shard might live on

That makes sense to me, I was just wondering about the name of the top-level key for the section (currently `reloaded_nodes`). The name seemed fitting when this section only contained node IDs, but now that it contains analyzer information as well, maybe there could be a better name? In Java this is called `reloadedIndicesDetails`, so I was bit surprised to see it called `reloaded_nodes` in the REST response as opposed to something like `reloaded_indices`.

> I know we use index names as keys in several places but if not necessary I consciously try to avoid it.

I agree it makes parsing more difficult! To me it would be best to maintain consistency across APIs (even if it makes parsing a little harder) as opposed to having a mix of styles, since this is least surprising to users. However this may be a bigger conversation, and the API is marked as experimental and can be changed, so it doesn't seem like it should prevent merging.

Reply:
I see what you mean now with the top-level key, sorry that I missed that part. Will change to something more neutral and probably also allign the java API to that new key a bit.